packages/text-editor/src/components/CollaborationUser.svelte

```diff
@@ -0,0 +1,62 @@
+<!--
+//
+// Copyright © 2024 Hardcore Engineering Inc.
+//
+// Licensed under the Eclipse Public License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License. You may
+// obtain a copy of the License at https://www.eclipse.org/legal/epl-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+-->
+
+<script lang="ts">
+  import contact, { Contact, PersonAccount } from '@hcengineering/contact'
+  import { Ref } from '@hcengineering/core'
+  import { CollaborationUser } from '../types'
+  import { Button, Component } from '@hcengineering/ui'
+  import { getClient } from '@hcengineering/presentation'
+
+  export let value: CollaborationUser
+  export let lastUpdate: number
+
+  const client = getClient()
+
+  let avatar: Component | undefined
+  $: lastUpdate !== 0 && avatar?.pulse()
+
+  let person: Contact | undefined
+  $: account = value.id as Ref<PersonAccount>
+  $: void fetchPerson(account)
```

Comment:
Why not use one of this?
```
export const employeeByIdStore = writable<IdMap<Employee>>(new Map())
export const employeesStore = writable<Employee[]>([])

export const personAccountByIdStore = writable<IdMap<PersonAccount>>(new Map())

export const channelProviders = writable<ChannelProvider[]>([])

export const personAccountPersonByIdStore = writable<IdMap<Person>>(new Map())
```

Reply:
because of circular dependency: contact-resources > view-resources > text-editor